### ISSUE TITLE
Drifters have super lasers that aren't very good 

### ISSUE THREAD

**Issue by JamesPagetButler** [aldrik/pa-mods#5]
Drifters can shoot sea floor but not subs

**Comment by aldrik** [comment]
Yes, it looks stupid and doesn't make any sense. There are technical issues that prevent fixing the cosmetics of this while keeping the current balance.
